io.github.pol_rivero.DesktopPlus.metainfo.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.pol_rivero.DesktopPlus</id>
+  <name>GitHub Desktop Plus</name>
+  <summary>GitHub Desktop fork with advanced functionality and Bitbucket integration</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <developer id="io.github.pol_rivero">
+    <name>Pol Rivero</name>
+  </developer>
+  <description>
+    <p>
+      This is a fork of shiftkey's GitHub Desktop port. In addition to being frequently updated
+      with the official upstream (desktop/desktop), it has many quality of life improvements and
+      advanced features for power users.
+    </p>
+    <p>
+      <h3>General:</h3>
```

Comment:
this is not html only the tags listed here are supported https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines#description